File: 167. Two Sum II - Input array is sorted/Python/Solution.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        i = 0
        j = len(numbers) - 1
        while i < j:
            two_sum = numbers[i] + numbers[j]
            if two_sum == target:
                break
            if two_sum > target:
                j -= 1
            else:
                i += 1
        return [i + 1, j + 1]

File: 167. Two Sum II - Input array is sorted/Java/Solution.java
public class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i = 0;
        int j = numbers.length - 1;
        while (i < j) {
            int twoSum = numbers[i] + numbers[j];
            if (twoSum == target) {
                break;
            }
            if (twoSum > target) {
                j--;
            } else {
                i++;
            }
        }
        return new int[] {i + 1, j + 1};
    }
}